### ISSUE TITLE
icu4c and zlib homebrew modules reinstalled on each boxen run

### ISSUE THREAD

**Issue by mattheath** [mattheath/puppet-php#13]
Both the icu4c and zlib homebrew modules seem to be reinstalled on every boxen run. The output log shows this:

```
Notice: /Stage[main]/Php/Package[icu4c]/ensure: created
Notice: Package[homebrew/dupes/zlib](provider=homebrew): Updating homebrew formulas
Notice: /Stage[main]/Php/Package[homebrew/dupes/zlib]/ensure: created
```

**Comment by boztek** [comment]
These are all keg only packages.

**Comment by mattheath** [comment]
Hmmm, guessing there's a separate boxen issue floating around somewhere - I would have expected the homebrew package provider to not repeatedly reinstall keg only packages.

**Comment by mattheath** [comment]
Should be fixed by boxen/puppet-boxen#41

**Comment by mattheath** [comment]
Fixed in version `2.3.2` of [boxen/puppet-boxen](https://github.com/boxen/puppet-boxen)
